src/Abp/Application/Features/NullFeatureValueStore.cs

```diff
@@ -12,7 +12,8 @@ public class NullFeatureValueStore : IFeatureValueStore
         /// <summary>
         /// Gets the singleton instance.
         /// </summary>
-        public static NullFeatureValueStore Instance { get { return SingletonInstance; } }
+        public static NullFeatureValueStore Instance => SingletonInstance;
```

Comment:
How about:

```c#
public static NullFeatureValueStore Instance { get; } = new NullFeatureValueStore();
```

There are many such cases — search for `Instance { get` in an IDE or text editor.
This kind of refactoring should be done across the entire repo in a single commit.

Reply:
@acjh Remove the unused code?
  `private static readonly NullFeatureValueStore SingletonInstance = new NullFeatureValueStore();`

Reply:
Oh, just saw that it's private.  I'll remove it.